test/logger.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env mocha */
+'use strict';
+const Winston = require('winston');
+
+describe('logging', () => {
+  it('returns a WINSTON object', () => {
+    const config = require('../lib/config');
+    const log = require('../lib/logger').attach(config);
+
+    log.should.be.an.instanceOf(Winston.Logger);
```

Comment:
Not sure I agree unless we're saying that we don't care what logging lib we choose as long as it has those messages. To me, what you're suggesting is effectively the same as testing Winston.

However, thinking about it, having a global logger might not be a bad idea and I'm definitely going to build a `Config`-like object to pass into the logging module for testing.